### ISSUE TITLE
Drawer Navigation does not have an option to configure spacing between Icon and Label

### ISSUE THREAD

**Issue by Pavelas** [react-navigation/react-navigation#9419]
**Current Behavior**
It seems that at the moment the margin between Icon and Label can't be configured, because of this part of code.

**Line: 183 from DrawerItem.tsx**
`
{ marginLeft: iconNode ? 32 : 0, marginVertical: 5 },
`

As the result, it becomes impossible to make the space smaller between icon and label.

**Expected Behavior**
My expected behavior would be to add possibility overwrite marginLeft inside labelStyle option.

**How to reproduce**
Nothing to reproduce.

**Your Environment**

| software                       | version |
| ------------------------------ | ------- |
| iOS or Android                 | Both
| @react-navigation/native       | 5.9.3
| @react-navigation/drawer       | 5.12.4
| react-native-reanimated        | 1.13.0
| react-native-gesture-handler   | 1.8.0
| react-native-safe-area-context | 3.1.9
| react-native-screens           | 2.15.2
| react-native                   | 0.64
| expo                           | 40.0.0
| node                           | v15.11.0
| npm or yarn                    | 7.6.0

**Comment by Rahulv2492** [comment]
@Pavelas you can use like that :
` options={{
        drawerLabel:()=><Text  style={{marginLeft:-20}}>Demo</Text>,
       drawerIcon: ....
      }}`

**Comment by Pavelas** [comment]
> @Pavelas you can use like that :
> ` options={{ drawerLabel:()=><Text style={{marginLeft:-20}}>Demo</Text>, drawerIcon: .... }}`

Yes, thank you very much! I am using it already, but I spent a lot of time before I found that there is marginLeft: 32, so I think it would be nice if we could override it intuitively.

**Comment by mattaningram** [comment]
Yeah this setup is strange, why can't we target the styling of the `View` around the label?

**Comment by vinamrahalgar** [comment]
Want to decrease the vertical padding when the label is selected. Any idea how to fix this
